bin/runsvdir-populate-service-dir

```diff
@@ -18,8 +18,12 @@ end
 
 File.open(PROCFILE_PATH) do |f|
   while line = f.gets
-    line.chomp!
```

Comment:
I think this should stay, otherwise `command` will end up with the newline in it. Not a huge deal but would make the `run` contents nicer.